data/rulesets.js

```diff
@@ -112,7 +112,7 @@ exports.BattleFormats = {
 				problems.push((set.name || set.species) + ' is higher than level 100.');
 			}
 
-			if (!allowCAP || template.tier !== 'CAP') {
+			if (!allowCAP || template.tier !== 'CAP' || template.tier !== 'CAP NFE' || template.tier !== 'CAP LC') {
```

Comment:
I'm not sure if I like hardcoding, if a format called CAP UU would be introduced, this code would have to be changed again. What about using `.startsWith` method?